README.md

```diff
@@ -64,9 +64,6 @@ Metadata contains three possible keys which may contain several possible values:
 `device`
```

Comment:
Need to change the line above that says "We do not yet support drives that use full-disk encryption with VeraCrypt." ;-)